### PR TITLE
fix(Dropdown): Rendering no matches message when all items selected

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Show no matches found message when all items are selected in the `Dropdown`

Screenshot 2020-05-12 at 10 02 20

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/13109)

### CODE REVIEW

File: packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -810,6 +810,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
 
   renderItemsListFooter(styles: ComponentSlotStylesInput) {
     const { loading, loadingMessage, noResultsMessage, items } = this.props;
+    const { value } = this.state;
```
Comment (pompomon) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:813]:
```suggestion
    const { filteredItems } = this.state;
```

File: packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -823,7 +824,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       };
     }
 
-    if (items && items.length === 0) {
+    if (items && (items.length === 0 || items.length === value.length)) {
```
Comment (pompomon) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:827]:
```suggestion
    if (filteredItems && filteredItems.length === 0) {
```

_nit_

Comment (pompomon) [reply packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:827]: I think it would work the same, and without additional condition

Comment (pompomon) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:827]: Please could you add a test for that new case?

Comment (assuncaocharles) [reply packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx]: Done

Comment (assuncaocharles) [reply packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx]: Done

**Comment by MaheshakaMudli** [comment]
I am facing the same issue. I am using the control in creatng teams messaging extension and the dropdown is not showing the "no matches" text when the user enters something or selects all the options from the dropdown and then searches for any text.
 
image

There is blank ul element created below the dropdown but it does not have any element created in it.

**Comment by assuncaocharles** [comment]
Hey @MaheshakaMudli 

Can you open a new issue and provide an codesandbox with small reproduction?

**Pull request merged**